langchain/callbacks/manager.py

```diff
@@ -45,6 +46,9 @@
 )
 from langchain.schema.messages import BaseMessage, get_buffer_string
 
+if TYPE_CHECKING:
```

Comment:
I could also jus type it as "any" if we want to avoid surfacing too much langsmith stuff in our high level files. I want to remove the langsmith dependency at some point anyway because it shouldn't be a hard req for people using langchain